src/language/__tests__/lexer-test.ts

```diff
@@ -1189,6 +1193,33 @@ describe('Lexer', () => {
   });
 });
 
+describe('SchemaCoordinateLexer', () => {
+  it('can be stringified', () => {
+    const lexer = new SchemaCoordinateLexer(new Source('Name.field'));
+    expect(Object.prototype.toString.call(lexer)).to.equal(
```

Comment:
This should be roughly equivalent?
```suggestion
    expect(String(lexer)).to.equal(
```

Reply:
deferring to prior art / established convention https://github.com/search?q=repo%3Agraphql%2Fgraphql-js%20Object.prototype.toString.call&type=code

(but I agree with you)